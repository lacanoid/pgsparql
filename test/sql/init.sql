CREATE EXTENSION plperl;
CREATE EXTENSION plperlu;
CREATE EXTENSION sparql;
\pset tuples_only

\dx sparql

create table ns1 as select name from sparql.namespace;

DROP extension sparql;

CREATE EXTENSION sparql VERSION '0.3';
ALTER EXTENSION sparql update;

\dx sparql

create table ns2 as select name from sparql.namespace;

-- namespaces in clean but not updated version
select name from ns1 where name not in (select name from ns2); 

-- namespaces in updated but not clean version
select name from ns2 where name not in (select name from ns1); 
