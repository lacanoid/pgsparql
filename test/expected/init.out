CREATE EXTENSION plperl;
CREATE EXTENSION plperlu;
CREATE EXTENSION sparql;
\pset tuples_only

\dx sparql
 sparql | 1.0     | sparql | SPARQL compiler


